<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2033</width>
    <height>1176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="AIO_ConfigurationAnalyseTool.qrc">
    <normaloff>:/new/prefix1/images/Portfolio.ico</normaloff>:/new/prefix1/images/Portfolio.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>291</width>
      <height>801</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>272</width>
       <height>1000</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1000</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2000</height>
      </size>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>138</width>
        <height>242</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_dev">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设备</string>
         </property>
         <property name="icon">
          <iconset resource="AIO_ConfigurationAnalyseTool.qrc">
           <normaloff>:/new/prefix1/images/device.png</normaloff>:/new/prefix1/images/device.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_config">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>配置</string>
         </property>
         <property name="icon">
          <iconset resource="AIO_ConfigurationAnalyseTool.qrc">
           <normaloff>:/new/prefix1/images/settings.png</normaloff>:/new/prefix1/images/settings.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_debug">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>调试</string>
         </property>
         <property name="icon">
          <iconset resource="AIO_ConfigurationAnalyseTool.qrc">
           <normaloff>:/new/prefix1/images/analyse.png</normaloff>:/new/prefix1/images/analyse.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_comm">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通信</string>
         </property>
         <property name="icon">
          <iconset resource="AIO_ConfigurationAnalyseTool.qrc">
           <normaloff>:/new/prefix1/images/communication.png</normaloff>:/new/prefix1/images/communication.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_realtime">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>实时数据</string>
         </property>
         <property name="icon">
          <iconset resource="AIO_ConfigurationAnalyseTool.qrc">
           <normaloff>:/new/prefix1/images/realtime.png</normaloff>:/new/prefix1/images/realtime.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_history">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>历史查询</string>
         </property>
         <property name="icon">
          <iconset resource="AIO_ConfigurationAnalyseTool.qrc">
           <normaloff>:/new/prefix1/images/history.png</normaloff>:/new/prefix1/images/history.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>1571</width>
      <height>801</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>3000</width>
       <height>2000</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>3000</width>
       <height>2000</height>
      </size>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>691</width>
        <height>561</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="page_debug">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>张力值：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>线速度：</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_config">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>91</x>
          <y>63</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>张力设定：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>209</x>
          <y>64</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>91</x>
          <y>133</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>线速度设定：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>140</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_comm">
       <widget class="QLabel" name="label_SendData">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>发送数据：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SendText">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>201</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_RecvData">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>150</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>接收数据：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_RecvText">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>140</y>
          <width>201</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Send">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>80</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Recv">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>180</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>接收</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page_dev">
       <widget class="QLineEdit" name="lineEdit_ConnState">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Connect">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Disconnect">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>130</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>63</x>
          <y>60</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>设备列表：</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_ConnState">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>920</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备连接状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentConnState">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>900</y>
      <width>44</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2033</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(F)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助(H)</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_help"/>
  </widget>
 </widget>
 <resources>
  <include location="AIO_ConfigurationAnalyseTool.qrc"/>
 </resources>
 <connections/>
</ui>
